core/domain/draft_upgrade_services.py

```diff
@@ -176,6 +176,22 @@ def extract_html_from_draft_change_list(draft_change_list):
 class DraftUpgradeUtil(python_utils.OBJECT):
     """Wrapper class that contains util functions to upgrade drafts."""
 
+    @classmethod
+    def _convert_states_v38_dict_to_v39_dict(cls, draft_change_list):
+        """Converts draft change list from state version 38 to 39. State
+        version 39 adds a customization arg for the Numeric Expression Input
+        interactions that allows creators to modify the placeholder text,
+        for which there should be no changes to drafts.
+
+        Args:
+            draft_change_list: list(ExplorationChange). The list of
+                ExplorationChange domain objects to upgrade.
+
+        Returns:
+            list(ExplorationChange). The converted draft_change_list.
+        """
+        return draft_change_list
```

Comment:
There can be changes in custarg for NumericInout interaction, right? Am I missing something?

Reply:
There were no cust args before this so there won't be any draft list changes, I believe.